source/_integrations/wake_on_lan.markdown

```diff
@@ -46,6 +46,15 @@ Sample service data:
 }
 ```
 
+<div class='note'>
+This usually only works if the Target Device is connected to the same network. Routing the WakeOnLan packet to a different subnet requires special configuration on your router or may not be possible.
+The Service to Route the packet is most likely named "IP Helper" which may support WakeOnLan, but not all Routers support this.
+</div>
+
+<div class='note'>
+On OPNsense you can work around this by installing the os-wol Package. WOL can then be triggered via [REST API](https://wiki.opnsense.org/development/api/plugins/wol.html) from Home Assistant, by using the [RESTful Command](https://www.home-assistant.io/integrations/rest_command) Integration.
+</div>
+
```

Comment:
This is not the OPNsense documentation.

We want to prevent documenting third-party workaround/solutions like this. The main reasoning is to prevent extensive documentation on all routers/firewalls out there (which would be a lot, hard to maintain and hard to read).

```suggestion
```